doc/source/getting_started/install.rst

```diff
@@ -393,6 +419,8 @@ pyreadstat                1.1.2              SPSS files (.sav) reading
 Access data in the cloud
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+Can be managed with the ``aws, gcp`` extras, depending on required compatibility.
```

Comment:
Would be good to add another column to specify which dependency is captured with which label.